build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 group 'com.github.romankh3'
-version '3.0.1'
+version '3.1.0-SNAPSHOT'
```

Comment:
Must be set 3.1.0 version instead and merge master branch and change in Pom.xml it.